src/main/kotlin/racingcar/model/racing.kt

```diff
@@ -0,0 +1,47 @@
+package racingcar.model
+
+import racingcar.view.printProgress
+
+
+const val START_RANGE = 0
+const val END_RANGE = 9
+const val TARGET_NUM = 4
+
+fun move(cars: MutableList<Car>) {
+    for (car in cars) {
+        val randomNum = rand()
+        car.position += goOrStop(randomNum)
+    }
+    printProgress(cars)
+}
+
+fun rand() = (START_RANGE..END_RANGE).random()
+
+fun goOrStop(num: Int): String {
+    if (num >= TARGET_NUM) return "-"
+    return ""
+}
+
+fun findWinnerNames(cars: List<Car>): List<String> {
+    var maxNum = -1
+    var winners = mutableListOf<String>()
+    for (car in cars) {
+        maxNum = winnerJudge(car, maxNum, winners)
+    }
+
+    return winners
+}
+
+private fun winnerJudge(
+    car: Car,
+    tmp: Int,
+    winners: MutableList<String>
+): Int {
+    val len = car.position.length
+    var num = Math.max(len, tmp)
+    if (tmp < num) {
+        winners.clear()
+        winners.add(car.name)
+    } else if (tmp == len) winners.add(car.name)
```

Comment:
```suggestion
    if (tmp < num) {
        winners.clear()
        winners.add(car.name)
    } else if (tmp == len) {
         winners.add(car.name)
    }
```
IDE 의 reformat 기능을 활용하여, 컨벤션 부분은 잘 맞춰주세요~!